app/src/main/java/com/github/se/polyfit/data/local/dao/MealDao.kt

```diff
@@ -37,6 +38,12 @@ interface MealDao {
     return meal?.toMeal()
   }
 
+  @Query("SELECT * FROM MEALTABLE WHERE createdAt >= :date ")
+  fun getMealsCreatedOnOrAfterDate(date: LocalDate): List<Meal>
+
+  @Query("SELECT * FROM MEALTABLE WHERE createdAt <= :date ")
```

Comment:
i think this should be `createdAt == :date`?

Reply:
Thank you for catching that. It is a typo!